components/runners/ambient-runner/ambient_runner/bridges/claude/operational_events.py

```
@@ -0,0 +1,141 @@
+        session_id = self._session_id
+
+        def _do_push() -> None:
+            client.session_messages.push(
+                session_id,
+                event_type=mapped_type,
+                payload=payload,
+            )
+
+        try:
+            await asyncio.get_running_loop().run_in_executor(None, _do_push)
+        except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)